support-frontend/assets/components/orderSummary/contributionsOrderSummaryContainer.tsx

```diff
@@ -45,10 +59,38 @@ export function ContributionsOrderSummaryContainer({
 }: ContributionsOrderSummaryContainerProps): JSX.Element {
 	const contributionType = useContributionsSelector(getContributionType);
 
-	const { currencyId } = useContributionsSelector(
+	const { countryId, currencyId } = useContributionsSelector(
 		(state) => state.common.internationalisation,
 	);
-	const selectedAmount = useContributionsSelector(getUserSelectedAmount);
+	const { productType } = useContributionsSelector(
+		(state) => state.page.checkoutForm.product,
+	);
+	const billingPeriod = (productType[0] +
+		productType.slice(1).toLowerCase()) as BillingPeriod;
+	const currentPrice = useContributionsSelector(getUserSelectedAmount);
+	const originalPrice = useContributionsSelector((state) =>
```

Comment:
A thought here on naming.

- We have `productPrice`, which is the price from the catalogue
- We have `productPriceWithPromo` which is a _lower_ price with a promotion applied (non-applicable to contributions)
- We have `productPriceWithContribution` which is a _higher_ price with an additional contribution added.

With these three values and the thresholds, we should have everything we need to create the UIs and functionality we want. It's an initial thought, let's pair on it next week.

Reply:
Yep happy to change this but as we havent introduced PromoCodes in this R I will do on next one.  It will be `currentPrice` = `productPriceWithPromo` & `originalPrice` = `productPriceWithPromo`. I don't think `productPriceWithContribution` is required here (its only associated with fixed pricing),

Reply:
First thing in 3rd PromoCode PR https://github.com/guardian/support-frontend/compare/main...three-tier-promocode-splus-visual-change